modules/broker-auth/src/main/java/io/ballerina/messaging/broker/auth/AuthInitException.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ *
+ */
+package io.ballerina.messaging.broker.auth;
+
+/**
+ * This Exception class represents initialization errors in Auth module.
+ */
+public class AuthInitException extends Exception {
+
+    public AuthInitException(String message) {
```

Comment:
Shall we remove the unused code?

Reply:
I added it because its likely to be used in future. Isn't it better to provide common constructors when adding a class even if they are not immediately used?

Reply:
removed